app/policies/application_policy.rb

```diff
@@ -0,0 +1,42 @@
+class ApplicationPolicy
+  attr_reader :user, :record
+
+  def initialize(user, record)
+    raise Pundit::NotAuthorizedError, "must be logged in" unless user
+
+    @user = user
+    @record = record
+  end
+
+  def index?
+    false
+  end
+
+  def show?
+    false
+  end
+
+  def create?
+    false
+  end
+
+  def new?
+    create?
+  end
+
+  def update?
+    false
+  end
+
+  def edit?
+    update?
+  end
+
+  def destroy?
+    false
+  end
```

Comment:
I have written [my personal opinion on those defaults already](https://github.com/maciejfedor/monte-cinema/pull/8#discussion_r976957422), so let me just copy it here:

I don't think I said that during our presentation - and I probably should have - but, and this is my personal bias based on my experience - you should not be defining defaults in `ApplicationPolicy` for any actions. I see three important reasons for that:
### It could hide errors making it harder to debug them, let me show you a simple example:

```ruby
class ExamplePolicy < ApplicationPolicy
  def imdex?
    complicated_function_to_check_access(user)
  end
end
```

Because of this simple typo, if I somehow miss it(and I've been there, done that) my controller will always prevent `index` action, as in `ApplicationPolicy` makes it return always false - and this means I will likely think that there is a bug in that `complicated_function_to_check_access` instead. Which might waste a lot of my time just because I have made a simple typo.

Now, what if this defaults were not defined? Well, I will get no method error - probably with message there `Did you mean: imdex?" - which will help me find that typo instantly.

### It might waste a lot of other  people time

If we have defaults, then when we just forget in rush to implement policy method for specific action in rush, well, as long as we don't have a test for that(which we should have, but hey, reality often is not what it was supposed to be 😄 ) we might miss that this is not working properly, now it will go through QA, QA will not know what is happening, so they will try to find any passing scenario to provide more information for us,  finally will just tell us that there is something wrong with authorization, as they just can't do that action, then we will need to check what is wrong, hopefully, we will quickly find the missing method and fix that. Without defaults, we would just get `NoMethodError`, and probably after seeing that QA will instantly check with us if we know what is wrong, as this is much more informative.

### In security-related topics,  it is often better to be explicit.

Defaults are often slow killing poison in cases like that. Because they exist, somebody might think that, "hey, let's just make `index` return always true, as that's always the case - after all we decide what to show on index with scope, so we will just return empty list in worst case scenario". And it might be true in 99% cases, but there will be that one case where at the end of the  day only admins have access to given resource, so somebody restricted access through action, not with scope. Now we might have opened that resource for everybody. Not so good I would say 😄 

But as I said, that just my personal bias 😉